job/helpers.go

```diff
@@ -71,6 +71,14 @@ func (j Job) InstanceID() string {
 	return ""
 }
 
+func (j Job) Rescue() bool {
```

Comment:
This could probably benefit from a comment. Not a part of this, but a lint check for comments on exported types would be ideal.